server/src/main/java/org/tbbtalent/server/api/admin/CandidateAdminApi.java

```diff
@@ -328,7 +328,7 @@ public String generateToken(@PathVariable("cn") String candidateNumber,
                                 @RequestParam(defaultValue = "false") boolean restrictCandidateOccupations,
                                 @RequestParam(defaultValue = "") List<Long> candidateOccupationIds) {
              CvClaims cvClaims = new CvClaims(candidateNumber, restrictCandidateOccupations, candidateOccupationIds);
-             String token = this.candidateTokenProvider.generateCvToken(cvClaims, 365L);
+             String token = candidateTokenProvider.generateCvToken(cvClaims, 365L);
```

Comment:
There was a merge conflict I had to resolve here with changes that Tim Hill checked in. All good now.